invenio_jobs/services/results.py

```diff
@@ -178,4 +180,19 @@ def to_dict(self):
             ].sort  # We want to keep the sort field of the last item to resume search from here with search_after
             if sort:
                 res["hits"]["sort"] = list(sort)
+            if hasattr(self._results, "_truncated") and self._results._truncated:
+                res["warnings"] = [
+                    {
+                        "message": _(
+                            "Too many log results returned (%(total)s). "
+                            "Only the most recent %(max)s results are shown. "
+                            "Please refine your search criteria to reduce the result size.",
```

Comment:
Unfortunately, we cannot "refine" the search criteria, as we are not searching. I would remove completely this last sentence.